Vaca.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjercicioDos
+{
+    internal class Vaca : Mamifero, Muerte
+    {    //se sobrescribe el metodo de mamimero amamantar
+        public override void amamanta()
+        {
+            Console.WriteLine("la vaca da leche");
+        }
+        //logica para calcular murte y pares
+        public string Morir(int nacimiento, int faleccimiento, string causa)
```

Comment:
Error on name faleccimiento please check it

Reply:
has been corrected